.circleci/local_publish_helpers.sh

```diff
@@ -59,7 +59,10 @@ function generatePkgCli {
   yarn --production
 
   # Optimize package size
-  yarn rimraf **/*.d.ts **/*.js.map **/*.d.ts.map **/README.md **/readme.md **/Readme.md **/CHANGELOG.md **/changelog.md **/Changelog.md **/HISTORY.md **/history.md **/History.md
+  yarn rimraf **/*.d.ts **/*.js.map **/*.d.ts.map
+  yarn rimraf **/README.md **/readme.md **/Readme.md
+  yarn rimraf **/CHANGELOG.md **/changelog.md **/Changelog.md
+  yarn rimraf **/HISTORY.md **/history.md **/History.md
```

Comment:
Changed because doing it all in one gave me "too many arguments" type errors on multiple platforms